tcks/microprofile-rest-client-reactive/pom.xml

```diff
@@ -81,6 +81,9 @@
                         <!-- TODO fix this-->
                         <exclude>org.eclipse.microprofile.rest.client.tck.jsonb.InvokeWithJsonBProviderTest</exclude>
 
+                        <!-- Missing implementation - See org.jboss.resteasy.reactive.common.jaxrs.RuntimeDelegateImpl.createEntityPartBuilder -->
+                        <exclude>**/EntityPartTest</exclude>
```

Comment:
Is it something we need to consider implementing? Question for both you and @geoand .

Reply:
What does this do @radcortez ?

Reply:
This tests the ` EntityPart.Builder`, which we don't have an implementation for in Quarkus REST:
https://github.com/quarkusio/quarkus/blob/07b3f949d3df35f796dc3124e1e3f72249fe1502/independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/jaxrs/RuntimeDelegateImpl.java#L145-L149

I'm not too concerned, especially because we can't use Quarkus REST to certify at this time. We could provide a proper implementation, but that can be done as a separate PR.

Reply:
Sure yeah, thanks